requirements.txt

```diff
@@ -1,3 +1,4 @@
 gevent==20.6.2
 gunicorn==19.9.0
-flask==1.1.1
\ No newline at end of file
+flask==1.1.1
+Pillow==6.2.2
```

Comment:
Version number might have to be adjusted to >=7.1.0 according to [this alert](https://github.com/geoadmin/mf-chsdi3/network/alert/requirements.txt/Pillow/open)

Reply:
updated to version 7.2.0